challenges/advanced-overload-default/question.py

```diff
@@ -0,0 +1,19 @@
+"""
+TODO:
+
+foo is a function that returns an interger when second argument is True, otherwise it returns a string.
+"""
```

Comment:
This seems duplicate of `advanced-overload`.

Reply:
overload-default -> overload-literal